conductor/lib/downloader.py

```diff
@@ -1270,23 +1270,32 @@ def make_url(path):
         '''
         TODO: get rid of this hardcoding!!!
```

Comment:
Actually, all it's trying to do is figure out which env (prod, qa, dev) the code is being run in. Maybe we should add something more explicit to the conductor config?